### DISCUSSION TITLE
Is there any way to modify the fixture already attached to a previous intercept?

### DISCUSSION THREAD

**Discussion by daniel-simoes-koerber** [cypress-io/cypress#17970] [Questions and Help] [answered]
My use case is: I have a popup that I want to test on my frontend - however, this will only appear if our backend sends a different response (any attribute on the response is changed). our frontend will be performing the same request from time to time (lets say every minute) and if something changed on the API response, then the popup will appear

currently this is what I have
```
const myMock= () => {
  cy.intercept(
    {
      method: "GET",
      path: "/notify"
    },
    {
      fixture: "my_fixture.json"
    }
  ).as(notification);
};
```

I have seen a response related to this but the solution is a bit cumbersome [stackoverflow](https://stackoverflow.com/questions/65044704/change-fixture-response-in-cypress-for-the-same-url-with-intercept)

Is there any better way to solve this?

**Comment by bahmutov** [comment, marked as answer]
Cypress command [cy.intercept](https://on.cypress.io/intercept) has the
 `times` parameter that you can use to create intercepts that only are used N times. In your case it would be 

```js
cy.intercept('http://localhost:4200/testcall', { 
  fixture: 'example.json',
  times: 1 
});
...
cy.intercept('http://localhost:4200/testcall', { 
  fixture: 'example2.json',
  times: 1
});
```

See the `cy.intercept` example in the Cypress recipes repo https://github.com/cypress-io/cypress-example-recipes#network-stubbing-and-spying

**Reply by daniel-simoes-koerber** [reply]
I also had this solution in my mind, like overriding the first intercept. I think it will be enough to solve my problem 
Thanks! :)
